lib/src/DatePicker/components/Calendar.tsx

```diff
@@ -5,7 +5,7 @@ import DayWrapper from './DayWrapper';
 import CalendarHeader from './CalendarHeader';
 import EventListener from 'react-event-listener';
 import SlideTransition, { SlideDirection } from './SlideTransition';
-import withStyles, { WithStyles } from '@material-ui/styles/withStyles';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
```

Comment:
You sure about that? I thought the main idea is to move all style stuff into own package.

Reply:
Thanks that’s seems to be missing :)